src/table/utils/__tests__/styling-utils.spec.ts

```diff
@@ -294,7 +297,7 @@ describe('styling-utils', () => {
     });
     // Only checking hover properties from here on
     it('should return styling with no hoverBackgroundColor and the specified hoverFontColor', () => {
-      layout.components[0].content.hoverFontColor.index = 1;
+      if (layout.components?.[0].content?.hoverFontColor) layout.components[0].content.hoverFontColor.index = 1;
```

Comment:
had to do a sanity check here since components, content and hoverFontColor can be undefined. Note that we only change the layout in the tests, so we don't have to deal with this check in that case.